src/mlpack/bindings/python/setup.py.in

```diff
@@ -42,7 +42,7 @@ else:
                   library_dirs=['${CMAKE_BINARY_DIR}/lib/'],
                   # CMAKE_CXX_FLAGS seems to have an extra space.
                   extra_compile_args=('-DBINDING_TYPE=BINDING_TYPE_PYX ' + \
-                      '-std=c++11${CMAKE_CXX_FLAGS}').split(' '),
+                      '-std=c++11 ${CMAKE_CXX_FLAGS}').split(' '),
```

Comment:
Thanks for the fix.  I found in my workings with the Python bindings that `CMAKE_CXX_FLAGS` always seemed to have an extra space.  But it seems like in your situation that isn't the case.  Could you test this and see if it works for you?  It should handle both cases:

```
('-std=c++11 ' + '${CMAKE_CXX_FLAGS}'.strip()).split(' '),
```

If that works for you then it will work in the cases I've tested too and I think we should go with that.